example/src/test/java/org/wordpress/android/fluxc/network/rest/wpcom/site/SiteRestClientTest.kt

```diff
@@ -526,6 +529,26 @@ class SiteRestClientTest {
         assertThat(options).containsEntry("site_creation_flow", siteCreationFlow)
     }
 
+    @Test
+    fun `when all domains are requested, then the correct response is built`() = test {
+        val allDomainsJson = "wp/all-domains/all-domains.json"
+        val json = UnitTestUtils.getStringFromResourceFile(javaClass, allDomainsJson)
+        val responseType = object : TypeToken<AllDomainsResponse>() {}.type
+        val response = GsonBuilder().create().fromJson(json, responseType) as AllDomainsResponse
+
+        initAllDomainsResponse(data = response)
+
+        val responseModel = restClient.fetchAllDomains(noWpCom = true)
+        assert(responseModel is Success)
+        with((responseModel as Success).data) {
+            assertThat(domains).hasSize(2)
+            assertThat(domains[0].domain).isEqualTo("some.test.domain")
+            assertThat(domains[0].wpcomDomain).isFalse()
+            assertThat(domains[1].domain).isEqualTo("some.test.domain 2")
+            assertThat(domains[1].wpcomDomain).isTrue()
```

Comment:
_Optional nitpick_: Android Studio recommends using property access syntax here instead of calling getters.
```suggestion
            assertThat(domains[0].wpcomDomain).isFalse
            assertThat(domains[1].domain).isEqualTo("some.test.domain 2")
            assertThat(domains[1].wpcomDomain).isTrue
```

Reply:
Good catch @justtwago 🙇 
I've committed the suggestion.